setup/base_dj/setup.py

```diff
@@ -2,7 +2,5 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon={'odoo_version_override': '10.0'},
-    include_package_data=True,
-    package_data={'': ['*', ]}
+    odoo_addon=True,
```

Comment:
Question: we are not forcing anymore the version 10.0 of Odoo?

Reply:
is not needed unless you want to override it. I was playing w/ setuptools-odoo w/ try n catch and I got this garbage. See https://github.com/OCA/website-cms/pull/34

Reply:
Ok, thanks for the information